Unstable Mongoose package

I've just cloned the repository, did `$ npm install` and tried to run everything with node foreman, and I've had a bunch of warnings about an unstable Mongoose version, resulting later in an undefined property error. This has blocked me from running the example application.

```
$ nf start
[WARN] No ENV file found
[OKAY] Trimming display Output to 118 Columns
8:42:46 AM web.1  |  { [Error: Cannot find module '../build/Release/bson'] code: 'MODULE_NOT_FOUND' }
8:42:46 AM web.1  |  js-bson: Failed to load c++ bson extension, using pure JS version
8:42:46 AM worker.1 |  { [Error: Cannot find module '../build/Release/bson'] code: 'MODULE_NOT_FOUND' }
8:42:46 AM worker.1 |  js-bson: Failed to load c++ bson extension, using pure JS version
8:42:46 AM web.1  |
8:42:46 AM web.1  |  ##############################################################
8:42:46 AM web.1  |  #
8:42:46 AM web.1  |  #   !!! MONGOOSE WARNING !!!
8:42:46 AM web.1  |  #
8:42:46 AM web.1  |  #   This is an UNSTABLE release of Mongoose.
8:42:46 AM web.1  |  #   Unstable releases are available for preview/testing only.
8:42:46 AM web.1  |  #   DO NOT run this in production.
8:42:46 AM web.1  |  #
8:42:46 AM web.1  |  ##############################################################
8:42:46 AM web.1  |
8:42:46 AM worker.1 |
8:42:46 AM worker.1 |  ##############################################################
8:42:46 AM worker.1 |  #
8:42:46 AM worker.1 |  #   !!! MONGOOSE WARNING !!!
8:42:46 AM worker.1 |  #
8:42:46 AM worker.1 |  #   This is an UNSTABLE release of Mongoose.
8:42:46 AM worker.1 |  #   Unstable releases are available for preview/testing only.
8:42:46 AM worker.1 |  #   DO NOT run this in production.
8:42:46 AM worker.1 |  #
8:42:46 AM worker.1 |  ##############################################################
8:42:46 AM worker.1 |
8:42:46 AM worker.1 |  { [Error: Cannot find module '../build/Release/bson'] code: 'MODULE_NOT_FOUND' }
8:42:46 AM worker.1 |  js-bson: Failed to load c++ bson extension, using pure JS version
8:42:47 AM web.1  |  { [Error: Cannot find module '../build/Release/bson'] code: 'MODULE_NOT_FOUND' }
8:42:47 AM web.1  |  js-bson: Failed to load c++ bson extension, using pure JS version
8:42:47 AM worker.1 |
8:42:47 AM worker.1 |  ##############################################################
8:42:47 AM worker.1 |  #
8:42:47 AM worker.1 |  #   !!! MONGOOSE WARNING !!!
8:42:47 AM worker.1 |  #
8:42:47 AM worker.1 |  #   This is an UNSTABLE release of Mongoose.
8:42:47 AM worker.1 |  #   Unstable releases are available for preview/testing only.
8:42:47 AM worker.1 |  #   DO NOT run this in production.
8:42:47 AM worker.1 |  #
8:42:47 AM worker.1 |  ##############################################################
8:42:47 AM worker.1 |
8:42:47 AM web.1  |
8:42:47 AM web.1  |  ##############################################################
8:42:47 AM web.1  |  #
8:42:47 AM web.1  |  #   !!! MONGOOSE WARNING !!!
8:42:47 AM web.1  |  #
8:42:47 AM web.1  |  #   This is an UNSTABLE release of Mongoose.
8:42:47 AM web.1  |  #   Unstable releases are available for preview/testing only.
8:42:47 AM web.1  |  #   DO NOT run this in production.
8:42:47 AM web.1  |  #
8:42:47 AM web.1  |  ##############################################################
8:42:47 AM web.1  |
8:42:47 AM worker.1 |  type=info msg="starting worker" concurrency=1
8:42:47 AM worker.1 |  type=info msg=connected service=rabbitmq
8:42:47 AM worker.1 |  type=info msg=connected service=mongodb
8:42:47 AM worker.1 |  d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\base.js:246
8:42:47 AM worker.1 |          throw message;
8:42:47 AM worker.1 |                ^
8:42:47 AM worker.1 |  TypeError: Cannot read property 'length' of undefined
8:42:47 AM worker.1 |      at processResults (d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\db.js:1581:31…
8:42:47 AM worker.1 |      at d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\db.js:1619:20
8:42:47 AM worker.1 |      at d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\db.js:1157:7
8:42:47 AM worker.1 |      at d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\db.js:1890:9
8:42:47 AM worker.1 |      at Server.Base._callHandler (d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\con…
8:42:47 AM worker.1 |      at d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\server.js:481:18 8:42:47 AM worker.1 |      at MongoReply.parseBody (d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\respons…
8:42:47 AM worker.1 |      at null.<anonymous> (d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\…
8:42:47 AM worker.1 |      at emit (events.js:107:17)
8:42:47 AM worker.1 |      at null.<anonymous> (d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\…
[DONE] Killing all processes with signal  null
8:42:47 AM worker.1 Exited Successfully
8:42:47 AM web.1  |  type=info msg="starting server" concurrency=1 thrifty=false timeout= busy_ms=
8:42:47 AM web.1  |  type=info msg=connected service=rabbitmq
8:42:47 AM web.1  |  type=info msg=connected service=mongodb
8:42:47 AM web.1  |  d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\base.js:246
8:42:47 AM web.1  |          throw message;
8:42:47 AM web.1  |                ^
8:42:47 AM web.1  |  TypeError: Cannot read property 'length' of undefined
8:42:47 AM web.1  |      at processResults (d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\db.js:1581:31…

8:42:47 AM web.1  |      at d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\db.js:1619:20
8:42:47 AM web.1  |      at d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\db.js:1157:7
8:42:47 AM web.1  |      at d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\db.js:1890:9
8:42:47 AM web.1  |      at Server.Base._callHandler (d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\con…

8:42:47 AM web.1  |      at d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\server.js:481:18
8:42:47 AM web.1  |      at MongoReply.parseBody (d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\respons…

8:42:47 AM web.1  |      at null.<anonymous> (d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\…

8:42:47 AM web.1  |      at emit (events.js:107:17)
8:42:47 AM web.1  |      at null.<anonymous> (d:\Repos\node-articles-nlp\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\…

8:42:47 AM web.1  Exited Abnormally
```

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "jade": "^1.5.0",
     "lodash": "^2.4.1",
     "logfmt": "^1.1.2",
-    "mongoose": "^3.8.13",
+    "mongoose": "~3.8.13",
     "mongoose-cache": "git://github.com/hunterloftis/mongoose-cache",
     "mongoose-timestamp": "^0.3.0",
     "morgan": "^1.2.0",

